tests/scss/others/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,145 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`functional_argument_on_same_lines_in_maps.scss format 1`] = `
+====================================options=====================================
+parsers: ["scss"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+//simple test
+$display-breakpoints: map-deep-merge(
+  (
+    "print-only": "only print",
+    "screen-only": "only screen",
+    "xs-only": "only screen and (max-width: #{map-get($grid-breakpoints, "sm") - 1})",
+    "sm-only": "only screen and (min-width: #{map-get($grid-breakpoints, "sm")}) and (max-width: #{map-get($grid-breakpoints, "md") - 1})",
+  ),
+  $display-breakpoints
+);
+
+
+//list test
+@each 
+$size 
+in 
+$sizes {
+.icon
+#{$size} 
+{
+border
+: 
+"#{$size + "px"}" 
+solid 
+red;
+}
+}
+
+@each 
+$size 
+in 
+$sizes {
+.icon-#{$size} 
+{
+border
+: 
+"#{$size + "px"}" 
+solid 
+red;
+}
+}
+
+//map test
+$font-weights
+: 
+("regular": 400
+, 
+"medium": 500, 
+"bold": 
+700);
+@each 
+$name, 
+$boldness 
+in 
+$icons {
+.text
+#{
+$name
+} {
+color: red;
+font-weight: 
+"#{$boldness}"
+}
+}
+
+$font-weights
+: 
+("regular": 400
+, 
+"medium": 500, 
+"bold": 
+700);
+@each 
+$name, 
+$boldness 
+in 
+$icons {
+.text-#{
+$name
+} {
+color: red;
+font-weight: 
+"#{$boldness}"
+}
+}
+=====================================output=====================================
+//simple test
+$display-breakpoints: map-deep-merge(
+  (
+    "print-only": "only print",
+    "screen-only": "only screen",
+    "xs-only": "only screen and (max-width: #{map-get($grid-breakpoints, "sm") - 1})",
+    "sm-only": "only screen and (min-width: #{map-get($grid-breakpoints, "sm")}) and (max-width: #{map-get($grid-breakpoints, "md") - 1})",
+  ),
+  $display-breakpoints
+);
+
+//list test
+@each $size in $sizes {
+  .icon #{$size} {
+    border: "#{$size + "px"}" solid red;
+  }
+}
+
+@each $size in $sizes {
+  .icon-#{$size} {
+    border: "#{$size + "px"}" solid red;
+  }
+}
+
+//map test
+$font-weights: (
+  "regular": 400,
+  "medium": 500,
+  "bold": 700,
+);
+@each $name, $boldness in $icons {
+  .text #{ $name } {
```

Comment:
`.icon-#{$size}` above didn't add space around

Reply:
@fisker and should not, you can write it in different ways (all valid):
```
.text foo { }

.text- foo {}

.text-foo {}

```

Reply:
I mean space inside `#{}`, this one has space around `$name`, but `.icon-#{$size}` don't.

Reply:
Yep, you are right

/cc @agamkrbit

Reply:
Looks like we are keeping spaces on stable.

**Prettier 2.1.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBiYASAhgAj-gwonAXx2AB0oDIAbCAJyRwbgBMBuKvEqq9YoOLYylavjqNmrTt1L9gQvNiWlycvJKYt2XcbyjoR68Zoj1tMvTxAAaEBAAOMAJbQAzslBYGDCAHcABR8ETxQsWn8sAE9PewAjBiwwAGs4GABlR2SXKABzZBgGAFc4ezgAW3j2NnYAGSx84qw8uAAxRgqsGFd85BAsYpgIOxAACxgK2gB1MZd4d2ywOAzQ+ZcAN3no-rB3OJBc9zgGGECkvK7kADMI4-sAK3cADwAhJNT0jKwKuDrcuA3O5lEBPZ4ZXJ5WhwACKxQg8CBtHuIGyDGODH67j2B0cDFyMGmLjYMDGyAAHAAGex4iDHaZJRz9PFwDEbQH2ACO8Pg5ycYQG7gAtFA4Ox2KNWNyXKxzi0rkhbsiQccKi5CiUVZDoXCEYDFcD7DAsPEiSSyUgAExGpIuWiQgDCEAqCpArIArKNiscAComsJKlEbUoASSgtVgGTA+OcAEFwxkYNFoUjjiQSEA)
```sh
--parser scss
```

**Input:**
```scss
#{$a                      } {
    color: red;
  }

#{                      $a} {
    color: red;
  }
#{           $a           } {
    color: red;
  }
#{$a} {
    color: red;
  }
```

**Output:**
```scss
#{$a } {
  color: red;
}

#{ $a} {
  color: red;
}
#{ $a } {
  color: red;
}
#{$a} {
  color: red;
}

```

Reply:
yeah.